uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss

```diff
@@ -1,3 +1,33 @@
+/* Schema */
+$dashboard-sidebar-width: 225px;
+$dashboard-sidebar-color: $brand-secondary;
```

Comment:
Isn't there a partial available that contains just variables where you could put these?